src/components/WalletSecurityOverlay/index.tsx

```diff
@@ -18,8 +18,8 @@ import { ChainId } from '@portkey/provider-types';
 import useBaseContainer from 'model/container/UseBaseContainer';
 import { PortkeyEntries } from 'config/entries';
 import { TargetScene } from 'pages/Guardian/GuardianManage/type';
-
-function AlertBody({ accelerateChainId }: { accelerateChainId: ChainId }) {
+// function AlertBody({ accelerateChainId }: { accelerateChainId: ChainId }) {
```

Comment:
does this line of code need to be kept?

Reply:
keep this line until test send token function.    if it is ok,  need to delete.